src/cowboy_swagger.erl

```diff
@@ -51,8 +51,8 @@
   #{parameter_definition_name() =>
       #{ type => binary()
        , items => #{ type => binary()
-                   , properties => property_obj()
-                   }
+                         , properties => property_obj()
+                         }
```

Comment:
What happened here?
Yeah, I know… this project needs a formatter… but we don't yet have a comma-first one 🤷🏻

Reply:
God knows. Fixed it.